src/caffeine/program_startup_s.F90

```diff
@@ -11,10 +11,15 @@
     if (prif_init_called_previously) then
        stat = PRIF_STAT_ALREADY_INIT
     else
-       allocate(current_team)
-       call caf_caffeinate(current_team%heap_mspace, current_team%heap_start, non_symmetric_heap_mspace, current_team%gex_team)
-       nullify(current_team%parent_team)
-       nullify(current_team%coarrays)
+       allocate(initial_team%info)
+       call caf_caffeinate( &
+          initial_team%info%heap_mspace, &
+          initial_team%info%heap_start, &
+          non_symmetric_heap_mspace, &
+          initial_team%info%gex_team)
+       nullify(initial_team%info%parent_team)
+       nullify(initial_team%info%coarrays)
```

Comment:
We want the above two component initializations (and eventually others) to appear every time Caffeine allocates a `team_data` (which will eventually appear in at least two different places, here and `prif_form_team`).

Wouldn't it make more sense to define initializers for these fields?